archive/j/javascript/exportExample.ts

```diff
@@ -0,0 +1,2 @@
+//exporting to importExample.ts
```

Comment:
We currently don't support the export and import examples. Any chance you'd be willing to put together some documentation together for these solutions? You can see some examples here: https://sample-programs.therenegadecoder.com/projects/

Reply:
Sure, I'll work on a brief documentation for import and export for JavaScript. Meanwhile I'll remove them from the PR.